src/components/DateRangePickerInput.jsx

```diff
@@ -296,8 +296,10 @@ export default withStyles(({ reactDates: { color, sizing } }) => ({
   },
 
   DateRangePickerInput__withBorder: {
-    border: `1px solid ${color.border}`,
```

Comment:
minor; but changing `border` might be a breaking change for those overriding it with CSS

Reply:
I thought in the most recent version of react-dates the classNames are scrambled, no?

Reply:
with the aphrodite interface sure, but i'm not sure about with the css interface

Reply:
I think anyone overriding the border will see the same effect they did before, so I'm not concerned.